datacenter/dtr/2.2/guides/admin/backups-and-disaster-recovery.md

```diff
@@ -37,17 +43,30 @@ command creates a backup of DTR:
 
 These files are added to a tar archive, and the result is streamed to stdout.
 
+Things DTR's backup command doesn't back up:
+
+* The image scanning database (if using image scanning)
+* Image contents
+* Users, orgs, teams
+
+There is no way to back up the image scanning database. You can re-download it
```

Comment:
Since you start the sentence with "there is no way you can backup", you're giving people a mini hearth attack.
Maybe

Once you restore DTR from a backup, you can download the latest vulnerability database.